### DISCUSSION TITLE
Welcome to the find-a-team discussions!

### DISCUSSION THREAD

**Discussion by q9f** [ethb3rlin/find-a-team#1] [Announcements]
# ETHBerlin³ Teamfinding 2022

This repository contains an issue tracker that allows teams to find hackers, and hackers to find a team.

## TL;DR

-   Looking for a team? Create an issue with `[LFT]`: [find-a-team/issues/new?template=hacker-wants-team.md](https://github.com/ethb3rlin/find-a-team/issues/new?template=hacker-wants-team.md)
-   Looking for hackers? Create an issue with `[LFH]`: [find-a-team/issues/new?template=team-wants-hacker.md](https://github.com/ethb3rlin/find-a-team/issues/new?template=team-wants-hacker.md)
-   Want to create a new team? Create an issue with `[NEW]`: [find-a-team/issues/new?template=idea-wants-team.md](https://github.com/ethb3rlin/find-a-team/issues/new?template=idea-wants-team.md)
-   Browse existing inserations: [ethb3rlin/find-a-team/issues](https://github.com/ethb3rlin/find-a-team/issues)
-   Connect with hackers on Matrix: [#find:tchncs.de](https://matrix.to/#/#find:tchncs.de)

![ETHBerlin logo](https://ethberlin.ooo/card.png)

You are looking for a hacker to help buidl your idea or you're looking for a team to join? You've come to the right place! Hacking together is much more fun, so let's find you a team.

## Hacker looking for a team

You **have skills** and **want to find a team**? Great:

1.  Open a new issue using the template `Hacker wants team`. Here you can introduce yourself and tell potential teams why you're awesome. Feel free to add or create labels that match your skills.
2.  Browse all open issues with the label `looking-for-hackers` or `creating-new-team`.
3.  If you find a team with an idea that excites you, start engaging with the team. You can just comment in the issue. You should reference the issue from step 1 here, so that the team can learn a bit more about you.
4.  When you found a team please close your issue so that people know you have found a team.

## Team looking for a hacker

You **have a team** and **want to find more hackers**? Amazing:

1.  Open a new issue using the template `Team wants hacker`. Here you can pitch what you're working on and tell everyone why they should buidl it.  Feel free to add or create labels that match your skills.
2.  Wait for hackers to approach you or browse through the hacker profiles (all issues with the label `looking-for-team`) and engage with them.
3.  When you found all the hackers you dreamed of and needed close your issue so that everyone knows you're set.

## Idea looking for a team

Some people have great ideas, but don't have a team yet to help them bring it into reality. No worries!

1.  Open a new issue using the template `Idea wants team`. Here you can pitch your idea and tell everyone why they should help you buidl it.  Feel free to add or create labels that match your skills.
2.  Wait for hackers to approach you or browse through the hacker profiles (all issues with the label `looking-for-team`) and engage with them.
3.  When you found all the hackers you dreamed of and needed close your issue so that everyone knows you're set.

## Communication

Interacting through issues might be great to find an initial match, but you probably want to take the conversation off GitHub at some point.  Join the hacker chat room on Matrix: [#find:tchncs.de](https://matrix.to/#/#find:tchncs.de)

You can also just directly leave your personal Matrix address in your tracking issue if you prefer.
